packages/editor/src/browser/editor-contribution.ts

```diff
@@ -138,14 +136,14 @@ export class EditorContribution implements FrontendApplicationContribution, Comm
         commands.registerCommand(EditorCommands.SHOW_ALL_OPENED_EDITORS, {
             execute: () => this.quickInputService?.open('edt ')
         });
-        const splitHandlerFactory = (splitMode: DockLayout.InsertMode): CommandHandler => new CurrentWidgetCommandAdapter(this.shell, {
-            isEnabled: title => title?.owner instanceof EditorWidget,
-            isVisible: title => title?.owner instanceof EditorWidget,
-            execute: async title => {
-                if (title?.owner instanceof EditorWidget) {
-                    const selection = title.owner.editor.selection;
-                    const newEditor = await this.editorManager.openToSide(title.owner.editor.uri, { selection, widgetOptions: { mode: splitMode, ref: title.owner } });
-                    const oldEditorState = title.owner.editor.storeViewState();
+        const splitHandlerFactory = (splitMode: DockLayout.InsertMode): CommandHandler => ({
+            isEnabled: () => !!this.editorManager.currentEditor,
+            execute: async () => {
+                const { currentEditor } = this.editorManager;
+                if (currentEditor) {
+                    const selection = currentEditor.editor.selection;
+                    const newEditor = await this.editorManager.openToSide(currentEditor.editor.uri, { selection, widgetOptions: { mode: splitMode } });
+                    const oldEditorState = currentEditor.editor.storeViewState();
```

Comment:
It looks like you've resolved the conflict here in favor of your code, but the changes made in the conflicting commit were significant. I think your change was made to ensure that this item would be visible in electron even if no editor was present at startup. To achieve that, please keep the old code, but remove the `isVisible` and `isEnabled`.